resources/views/topic/reply.twig

```diff
@@ -9,7 +9,7 @@
 		<header>
 			<h1>{{ trans('topic.postreplytotopic', {'title':topic.title}) }}</h1>
 		</header>
-		{{ form_open({'route': ['topics.reply', topic.slug, topic.id], 'method': 'post', 'id': 'topicreply'}) }}
+		{{ form_open({'route': ['topics.reply', topic.id, topic.slug], 'method': 'post', 'id': 'topicreply'}) }}
```

Comment:
Hi,

It's better to use `{id: topic.id, slug: topic.slug}` instead, rather than positional arguments.